src/test/java/redis/clients/jedis/JedisClientSideCacheTest.java

```diff
@@ -0,0 +1,70 @@
+package redis.clients.jedis;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InOrder;
+import org.mockito.Mockito;
+
+public class JedisClientSideCacheTest {
+
+  protected static final HostAndPort hnp = HostAndPorts.getRedisServers().get(1);
+
+  protected Jedis jedis;
+
+  @Before
+  public void setUp() throws Exception {
+    jedis = new Jedis(hnp, DefaultJedisClientConfig.builder().timeoutMillis(500).password("foobared").build());
```

Comment:
glad to see this test

Reply:
this is our regular Jedis object, to verify and cleanup stuff